store/queries/post_hotness.sql

```diff
@@ -0,0 +1,24 @@
+-- name: DeleteOldPostHotness :execrows
+DELETE FROM post_hotness
+WHERE generated_at < NOW() - sqlc.arg(retention_period)::INTERVAL;
+
+-- name: MaterializeClassicPostHotness :one
```

Comment:
So two thoughts on this:
- Just wanted to double check, but these will all be inserted/show at once in the database?
- Do we know roughly what the performance/resource consumption of this will be like ? I can maybe run this as a test query but I'm worried about this potentially timing out.